app/controllers/users_controller.rb

```diff
@@ -106,8 +107,13 @@ def user_params
       *EmailPreferences::ATTRIBUTES,
       :application_learning_history, :application_skills, :application_code_samples,
       :application_location, :application_minimum_money, :application_money, :application_goals, :application_code_background,
+      :postal_addresses,
       interested_in: [],
       roles_attributes: [:id, :name, :team_id, :_destroy]
     )
   end
+
+  def address_params
```

Comment:
See my comment above in the `update` method: I think using Rails' `nested_attributes` would be a even better fit here. Just check the Rails guides on this, it's should be pretty well documented there. Ping me if you need any help or guidance 👋